oneBarangay/static/assets/js/volt.js

```diff
@@ -17,341 +17,339 @@
-    }
-
-    // Glide JS
-
-    if (d.querySelector('.glide')) {
-        new Glide('.glide', {
-            type: 'carousel',
-            startAt: 0,
-            perView: 3
-          }).mount();
-    }
-
-    if (d.querySelector('.glide-testimonials')) {
-        new Glide('.glide-testimonials', {
-            type: 'carousel',
-            startAt: 0,
-            perView: 1,
-            autoplay: 2000
-          }).mount();
-    }
-
-    if (d.querySelector('.glide-clients')) {
-        new Glide('.glide-clients', {
-            type: 'carousel',
-            startAt: 0,
-            perView: 5,
-            autoplay: 2000
-          }).mount();
-    }
-
-    if (d.querySelector('.glide-news-widget')) {
-        new Glide('.glide-news-widget', {
-            type: 'carousel',
-            startAt: 0,
-            perView: 1,
-            autoplay: 2000
-          }).mount();
-    }
-
-    if (d.querySelector('.glide-autoplay')) {
-        new Glide('.glide-autoplay', {
-            type: 'carousel',
-            startAt: 0,
-            perView: 3,
-            autoplay: 2000
-          }).mount();
-    }
+  if (d.querySelector(".ct-chart-traffic-share")) {
+    const data = {
+      series: [70, 20, 10],
+    };
 
-    // Pricing countup
-    var billingSwitchEl = d.getElementById('billingSwitch');
-    if(billingSwitchEl) {
-        const countUpStandard = new countUp.CountUp('priceStandard', 99, { startVal: 199 });
-        const countUpPremium = new countUp.CountUp('pricePremium', 199, { startVal: 299 });
-        
-        billingSwitchEl.addEventListener('change', function() {
-            if(billingSwitch.checked) {
-                countUpStandard.start();
-                countUpPremium.start();
-            } else {
-                countUpStandard.reset();
-                countUpPremium.reset();
-            }
-        });
-    }
+    const sum = function (a, b) {
+      return a + b;
+    };
 
-});
\ No newline at end of file
+    new Chartist.Pie(".ct-chart-traffic-share", data, {
+      labelInterpolationFnc: function (value) {
+        return Math.round((value / data.series.reduce(sum)) * 100) + "%";
+      },
+      low: 0,
+      high: 8,
+      donut: true,
+      donutWidth: 20,
+      donutSolid: true,
+      fullWidth: false,
+      showLabel: false,
+      plugins: [Chartist.plugins.tooltip()],
+    });
+  }
+
+  if (d.getElementById("loadOnClick")) {
+    d.getElementById("loadOnClick").addEventListener("click", function () {
+      const button = this;
+      const loadContent = d.getElementById("extraContent");
+      const allLoaded = d.getElementById("allLoadedText");
+
+      button.classList.add("btn-loading");
+      button.setAttribute("disabled", "true");
+
+      setTimeout(function () {
+        loadContent.style.display = "block";
+        button.style.display = "none";
+        allLoaded.style.display = "block";
+      }, 1500);
+    });
+  }
+
+  const scroll = new SmoothScroll('a[href*="#"]', {
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/PrynsTag/oneBarangay/pullRequest?prid=7988857)
```suggestion
  const scroll = new SmoothScroll("a[href*=\"#\"]", {
```